{{cookiecutter.project_shortname}}/docker/wait-for-services.sh

```diff
@@ -24,10 +24,10 @@ _db_check(){ docker-compose exec db bash -c "mysql -p{{cookiecutter.project_shor
 check_ready "MySQL" _db_check
 {%- endif %}
 
-_es_check(){ [[ $(curl -sL -w "%{http_code}\\n" "http://localhost:9200/" -o /dev/null)==200 ]]; }
+_es_check(){ curl -sL -w "%{http_code}\\n" "http://localhost:9200/" -o /dev/null | grep '200' &> /dev/null; }
```

Comment:
@egabancho I have changed this since it was not working for me (I was running it at home with a Linux machine), WDTY?